src/Illuminate/Database/Schema/Grammars/SqlServerGrammar.php

```diff
@@ -312,6 +312,22 @@ public function compileDisableForeignKeyConstraints()
         return 'EXEC sp_msforeachtable "ALTER TABLE ? NOCHECK CONSTRAINT all";';
     }
 
+    /**
+     * Compile the command to drop all foreign keys.
+     *
+     * @return string
+     */
+    public function compileDropAllForeignKeys()
+    {
+        return "DECLARE @sql NVARCHAR(MAX) = N'';
+            SELECT @sql += 'ALTER TABLE ' + QUOTENAME(OBJECT_SCHEMA_NAME(parent_object_id))
```

Comment:
Do we need to specify the database name?

Reply:
@staudenmeir I dont believe so because it's being called the same was as compileDropAllTables using $this->connection.
```
$this->connection->statement($this->grammar->compileDropAllForeignKeys());
$this->connection->statement($this->grammar->compileDropAllTables());
```

Reply:
When I ran this code, it only removed the foreign keys on the one database (I checked the other one on the same server, and it's foreign keys are still intact).